### PR TITLE
Create fellowship certificate page

### LINKED ISSUE #747: Create a new page to render fellowship certificates
**Is your feature request related to a problem? Please describe.**
Need to render an HTML generated certificate under `sefglobal.ord/fellowship`

**Describe the solution you'd like**
Create a new HTML page and render the certificate object using mustache.js 

**Additional context**
Design and other details can be found [here](https://sef.discourse.group/t/developing-an-admin-platform-for-sef-fellowship-program/20?u=gravewalker).

### PR DESCRIPTION
**Pull request by Gravewalker666**
## Purpose
The purpose of this PR is to fix #747 

## Goals

To make a page to function as the certificate view of fellowship program

## Approach

* Created a new folder `fellowship`
* Created a new HTML file under that.
* Used ajax to get the certificate object depending on the URL search param, `id`
  * `id` should be encoded in base64
* Rendered the object using mustache js 

### Screenshots

image

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Test environment
 
TestServer: Webpack dev-server
Os: MacOs Catalina 10.15.3
Test: Google Chrome

### CODE REVIEW

File: src/main/webapp/fellowship/index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    </div>
+</script>
+<script>
+    $(function () {
+        let searchParams =  new URLSearchParams(window.location.search)
+        if(searchParams.has('id')){
+            let id = atob(searchParams.get('id'));
+            $.ajax({
+                type: 'get',
+                url: 'http://localhost:8080/core/fellowship/certificates/'+id,
```
Comment (jayasanka-sack) [inline src/main/webapp/fellowship/index.html]: This URL should be changed

File: src/main/webapp/fellowship/styles.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.margin-left-0{
+    margin-left: 0 !important;
+}
+.min-h-100vh{
```
Comment (jayasanka-sack) [inline src/main/webapp/fellowship/styles.css:5]:
```suggestion

.min-h-100vh{
```

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/fellowship/styles.css (hunk 2 above)
Comment (Gravewalker666) [reply src/main/webapp/fellowship/styles.css:5]: On it

File: src/main/webapp/fellowship/index.html (hunk 1 above)
Comment (Gravewalker666) [reply src/main/webapp/fellowship/index.html]: Sorry, My mistake. Let me fix it.

Comment (Gravewalker666) [reply src/main/webapp/fellowship/index.html]: Resolved with commit https://github.com/sef-global/sef-site/pull/748/commits/f2d6f67a13c6bbc62464767011b16e79e033fa6b

File: src/main/webapp/fellowship/styles.css (hunk 2 above)
Comment (Gravewalker666) [reply src/main/webapp/fellowship/styles.css:5]: Resolved with commit https://github.com/sef-global/sef-site/pull/748/commits/f2d6f67a13c6bbc62464767011b16e79e033fa6b

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Gravewalker666 !!!

**Review by Piumal1999 — APPROVED** [review]
Well done!!

**Pull request merged**